.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+Welcome to Cryostat! Before contributing, make sure to:
```

Comment:
Any template to prefill the PR title too?

Something like: <type>[optional scope]: <description>

Reply:
Might not be possible at this time. https://github.com/isaacs/github/issues/1252 for example